nas.py

```diff
@@ -192,16 +192,21 @@ def get_middle_box_iface_name_by_network_name(self, host_name, network):
               "host {} and network name {}\n".format(host_name, network))
         sys.exit(1)
 
-    def get_all_iface_names(self, host_name):
+    def get_all_test_iface_names(self, host_name):
```

Comment:
the naming is a little bit strange. I know a differentiation between data and control (e.g. data plane, control plane). I think we should substitute every test with data. Simple search and replace ...